src/tripCard/index.tsx

```diff
@@ -75,14 +76,27 @@ const TripCard = ({
             className="kirk-tripCard-itinerary"
             places={itinerary}
           />
-          <span className="kirk-tripCard-price">{price}</span>
+          <Text className="kirk-tripCard-price" display={TextDisplayType.TITLESTRONG}>
+            {price}
+          </Text>
         </div>
         <div className="kirk-tripCard-bottom">
           <div className="kirk-tripCard-driver">
-            <Avatar image={driver.avatarUrl} alt={driver.firstName} />
-            <span>{driver.firstName}</span>
+            <div className="kirk-tripCard-avatar">
+              <Avatar
+                image={driver.avatarUrl}
+                alt={driver.firstName}
```

Comment:
nit: not sure the explicit `alt` is needed since the image is followed by the same text

Reply:
It's not needed, unless you want to precise more information than the driver's name.

Reply:
Yes that's what I meant, here the text is the same.